snippets/ci-shell/ci-shell.conf

```diff
@@ -0,0 +1,13 @@
+#
```

Comment:
This is specifically for Thread CI, right? If so, it's worth naming it more distinctively like "thread-ci-shell" or sth like that when put in the common snippet directory.

Reply:
it's related to CI. not necessarly to thread one. @tomaszkob89 is there any thing specific for Thread in here?

Reply:
No, I think it should be applied to all the CI samples